source/numerics.tex

```diff
@@ -6874,7 +6874,7 @@
 
 \begin{itemdescr}
 \pnum
-\effects Same as \tcode{valarray(il.begin(), il.size())}.
+\effects Equivalent to \tcode{valarray(il.begin(), il.size())}.
```

Comment:
We have been reluctant to add "Equivalent to"s in the past, since "Equivalent to" also notionally populates the other elements of the description whereas "Same as" does not; see [structure.specifications]/4.

@jwakely Can you check that the change to "Equivalent to" here doesn't have an (unintended) normative effect?